docs/reviewing-tests/index.md

```diff
@@ -52,9 +52,8 @@ requests.  If you are interested in receiving notifications of proposed
 changes to tests in a given directory, feel free to add yourself to the
 META.yml file. Anyone with expertise in the specification under test can
 approve a pull request.  In particular, if a test change has already
-been adequately reviewed "upstream" in another repository, it can be
-pushed here without any further review by supplying a link to the
-upstream review.
+been accepted in a trusted fork of WPT, it can be pushed here without any
```

Comment:
I think this is more confusing than what we had before. I actually don't think the policy limits the reviews to "trusted" forks, only requiring that the review is public.

So, maybe:

It is not required that the review is done in the wpt GitHub repository, as long as the review is done in public.
In particular, several browser projects have set up processes whereby changes made and reviewed in their own review systems are automatically synchronized with this repository, not requiring another review.